crates/controller/src/hud/map.rs

```diff
@@ -0,0 +1,30 @@
+use bevy::prelude::{default, AlignSelf, Color, Component, NodeBundle, Size, Style, Val};
+
+pub(crate) struct MapBundle {
+    background: Color,
+}
+
+impl MapBundle {
+    pub(crate) fn new(color: Color) -> MapBundle {
+        MapBundle { background: color }
+    }
+
+    pub(crate) fn build(&self) -> NodeBundle {
+        NodeBundle {
+            style: Style {
+                size: Size {
+                    width: Val::Percent(100.),
+                    height: Val::Percent(100.),
+                },
+                align_self: AlignSelf::Stretch,
```

Comment:
Alternatively, do not pass `size` and `align_self` but instead set `position: UiRect::all(Val::Percent(0.))`, something like that should work.

Reply:
Based on `mapselection` in a `menu` crate there is at least one more parameter needed. Having either `align_self` or as advised `position` works..